openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -597,6 +618,233 @@ Location: https://client.example.net/cb?
+       |  (9) Credential Request      |                        |                           |
+       |      (Access Token, proof(s))|                        |                           |
+       |------------------------------------------------------>|                           |
+       |                              |                        |                           |
+       |  (10) Credential Response    |                        |                           |
+       |       with Credential(s) OR  |                        |                           |
+       |       Transaction ID         |                        |                           |
+       |<------------------------------------------------------|                           |
+~~~
+!---
+Figure: Issuance using the Interactive Authorization Endpoint
+
+## Interactive Authorization Request {#interactive-authorization-request}
+
+The request to the Interactive Authorization Endpoint is formed and sent in the same way as PAR request as defined in [@!RFC9126, Section 2.1]. The contents of the request are the same as in a regular Authorization Request as defined in (#credential-authz-request), with the following additions:
+
+ - In case the Wallet has received an Interactive Authorization Response previously, the `auth_session` parameter from that response MUST be included in all subsequent requests (see (#iar-interaction-required-response)).
+ - In case the Wallet has completed a Presentation, it has to include the received redirect URI in the parameter `openid4vp_redirect_uri` (see (#iar-require-presentation)) during the next call to the Interactive Authorization Endpoint.
+
+Note: In case a Wallet Attestation is required by the Authorization Server, it has to be included in this request.
+
+## Interactive Authorization Response
+
+Upon receiving an Interactive Authorization Request, the Authorization Server determines whether the authorization request is semantically correct and valid, and whether the information provided by the Wallet so far is sufficient to grant authorization for the Credential issuance.
+The response to an Interactive Authorization Request is an HTTP message with the content type `application/json` and a JSON document in the body that indicates
+
+- an error as defined in Section 2.3 of [@!RFC9126], or
+- that user interaction is required, as defined in (#iar-interaction-required-response), or
+- a successful completion of the authorization, as defined in (#iar-authorization-code-response).
+
+Depending on this assessment, the response from the Interactive Authorization Endpoint can take one of the following forms:
+
+### Interaction Required Response {#iar-interaction-required-response}
+
+The Authorization Server MAY request an additional user interaction by sending a JSON body containing the following keys:
+
+* `status`: REQUIRED. MUST contain the string `require_interaction`, indicating that additional interaction is required.
```

Comment:
If this is always required and the value is always set to `require_interaction` then perhaps we can just remove this from the response as its implicit.

```suggestion
```

Otherwise we probably need to re-phrase it so that there can be other possible statuses in future.

Reply:
I think the real solution is to move "status" one subsection higher and define it there.